.github/workflows/run-checker-ci.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           enable-trace enable-fips,
           no-ts,
           no-ui,
+          enable-ktls no-ec,
```

Comment:
I'm not sure this is a good idea. Clearly this would have caught this problem, but there is a potential combinatorial explosion here. Why stop at the combination of "enable-ktls and no-ec"? We could keep going with all the other 2 option combinations (and why stop at 2?). This doesn't seem viable.

Reply:
One idea would be to enable all non-default options in these runchecker builds and disable just one enabled by default.

Reply:
This should cover most of the issues IMO.

Reply:
A more general purpose test is likely out-of scope for this one particular fix, and could discover a number of issues that cause this to blow up. Perhaps an issue to report what the found problems are, so they can be fixed? Whatever is done should probably not enable features that are dependent on other libraries (e.g. compression options).

Should I just delete this test for now?